packages/browser/src/Component/Itowns/3DTiles/3DTilesUtils.js

```diff
@@ -1,4 +1,145 @@
 import * as THREE from 'three';
+import { TilesManager } from './TilesManager';
+
+/**
+ * Create a geometric outliers for each tile of a tileset when its geometry is loaded
+ * Each outlier tile geometry does not keep the batchid for each outliers.
+ * May be slow to create / load.
+ *
+ * @param {TilesManager} tilesManager A TilesManage object from UD-Viz.
+ */
+export function createOutliersOfTileset(tilesManager) {
+  tilesManager.addEventListener(
+    TilesManager.EVENT_TILE_LOADED,
+    function (tile) {
+      if (tilesManager.tiles[tile.tileId] != undefined) {
+        if (tilesManager.tiles[tile.tileId].cityObjects != undefined) {
+          const geom =
+            tilesManager.tiles[tile.tileId].getObject3D().children[0]
+              .children[0];
+          if (!geom.hasOutlier) {
+            // This event can be triggered multiple times, even when the geometry is loaded.
+            // This bool avoid to create multiple outliers for one geometry
+            geom.hasOutlier = true;
+
+            // THREE.EdgesGeometry needs triangle indices to be created.
+            // Create a new array for the indices
+            const indices = [];
+
+            // Iterate over every group of three vertices in the unindexed mesh and add the corresponding indices to the indices array
+            for (
+              let i = 0;
+              i < geom.geometry.attributes.position.count;
+              i += 3
+            ) {
+              indices.push(i, i + 1, i + 2);
+            }
+            geom.geometry.setIndex(indices);
+
+            // Create Outliers
+            const geo = new THREE.EdgesGeometry(geom.geometry, 30); // or WireframeGeometry
```

Comment:
Apparently there is a WireframeGeometry
[ref](https://discourse.threejs.org/t/proper-way-of-adding-and-removing-a-wireframe/4600)

Reply:
WireframeGeometry will show each triangle of a mesh.
Here, I wanted to only make the edges of the mesh, not all triangles. We can also chose what define an edge : 
thresholdAngle — An edge is only rendered if the angle (in degrees) between the face normals of the adjoining faces exceeds this value. default = 1 degree.
(ref)[https://threejs.org/docs/#api/en/geometries/EdgesGeometry]

Reply:
ok got it, could be nice to make the thresholdAngle a parameter of the function with a default value at 30